src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -181,6 +181,11 @@ def CloseBLEConnection(self):
             lambda: self._dmLib.pychip_DeviceCommissioner_CloseBleConnection(self.devCtrl)
         )
 
+    def CloseSession(self, nodeid):
```

Comment:
Could you add test call in automatic script https://github.com/project-chip/connectedhomeip/blob/master/src/controller/python/test/test_scripts/mobile-device-test.py 
so that cirque presumit test could run it automatically, thanks

Reply:
Hmm. I'm happy to add a test, but I believe we don't have any cirque test cases which open a CASE connection, am I wrong?

Reply:
CloseSession means Close Case Session?

If CloseSession means close any PASE/CASE session, 
Possibly three test cases:
1. For PASE you can directly add this Close upon the existing test, where you can create pase session, close pase session, re-create pase session, in this PR, we could add this,

2. For CASE, I think it should be straightforward to add CASE connection test, let's discuss in slack how to do that, we can have the follow-up ticket

3. For CASE + Reboot, base upon test 2, we can issue command to reboot cirque docker, and simulate this situation, we could have a follow-up ticket

Reply:
Ok, added `CloseSession` at the end of the cirque test. Unfortunately, I'm not able to test the session resumption, because:
1. Currently, `zcl` commands fail when a device is not securely connected although they trigger a CASE session establishment in such a case. So they must be preceded with a call to `ResolveNode`.
2. `ResolveNode` would fail because Thread implementation for Linux (which is used by the cirque tests) does not have the SRP client support yet.
3. The SRP client support cannot be added until https://github.com/openthread/ot-br-posix/issues/855 is resolved.

Reply:
thanks, we would handle Add DBus API for register SRP service openthread/ot-br-posix#855  and enable SRP client asap.